### DISCUSSION TITLE
Finding template listing: custom display fields

### DISCUSSION THREAD

**Discussion by noraj** [Syslifters/sysreptor#118] [General]
All finding templates are listed under `/templates/`.

Currently, for each finding template there are the following information displayed:

- 1st row: CVSS score, title
- 2nd row: status, languages, tags

It would be nice if we could configure that (user settings or on the page directly) if one want to display OWASP category or the reference-id as well or even hide some fields.

![image](https://github.com/Syslifters/sysreptor/assets/16578570/f15023c8-bff4-40b5-ab59-5ef888cff388)

**Comment by silent6trinity** [comment]
I concur with this - personally, i'd like to customize the tags - like web/infrastructure ones without having to alter the software's codebase itself. Not a big deal though

**Reply by MWedl** [reply]
You can actually use custom tags. Select the tag field and type your custom tag names. The predefined tags in the selection are only suggestions of commonly used tags.

Maybe we could add a hint to the tag input field that you can also enter custom tags.

**Comment by Alex6440** [comment]
Is it possible to add sorting functions to the list of templates ? For example, according to status, tags, etc.

It might also be interesting to have a counter to keep track of the number of templates created.

Thanks :)

**Reply by aronmolnar** [reply]
Thanks Alex for the suggestions.

The sorting would be quite complex because the translations in the finding templates are nested fields, so there is more than one status, title, etc. per finding template.  
The cost/benefit is for this feature is currently too low.

We can add the number how many templates there are. This also requires adaptions in the pagination mechanism, but is doable.

Instead of sorting we rather want to improve filtering mechanisms, so you can filter, e.g. for a status. We have some open tickets about this already.
